### DISCUSSION TITLE
Waiting >3 days for a response from Enterprise Support

### DISCUSSION THREAD

**Discussion by leepa** [community/community#40058] [Enterprise]
### Select Topic Area

Product Feedback

### Body

Not sure if anyone else has suffered from this issue, and apologies for ranting here, but the radio silence from GitHub is irritating at this point. We are a GitHub Enterprise customer, and I raised a ticket three days ago about dealing with an LFS error (GH009) which tells me to contact support.

I've tried every solution to the error I can find on Google (admittedly not much), but ultimately if GitHub says to contact support, it would be good for them to answer.

How long do people wait for answers from support? It leaves a bitter taste when one pays for a service and doesn't get any answers...

**Comment by GrantJacobsSA** [comment]
I'm getting the same error and same results from you about not finding much on Google. Hopefully we can get an answer soon if you haven't gotten one yet.
